doc/project/project_roles.rst

```diff
@@ -314,15 +314,47 @@ Release Activity
 Merge Criteria
 ++++++++++++++
 
-* All continuous integration checks have passed
+* Minimal of 2 approvals, including an approval by the designated assignee.
+* Pull requests should be reviewed by at least a maintainer or collaborator of
+  each affected area; Unless the changes to a given area are considered trivial
+  enough, in which case approvals by other affected subsystems
+  maintainers/collaborators would suffice.
+* Four eye principle on the organisation level. We already require at least 2
+  approvals (basic four eye principle), however, such reviews and approvals
+  might be unintentionally biased in the case where the submitter is from the
+  same organisation as the approvers. To allow for project wide review and
+  approvals, the merge criteria is extended with the guidelines below:
+
+  * Changes or additions to common and shared code shall have reviews from
+    different organisations (at least one review and approval from an
+    organisation different than the submitters').
+    Common and shared code is defined as anything that does not fall under
+    :file:`soc`, :file:`boards` and :file:`drivers/*/*`.
+  * Changes or additions to hardware support (driver, SoC, boards) shall at
+    least have the merger be from a different organisation. This applies only
+    to implementation of an API supporting vendor specific hardware and not the
+    APIs.
+  * Release engineers shall not merge code changes originating and reviewed
+    only by their own organisation. To be able to merge such changes, at least
+    one review shall be from a different organisation.
+  * Release engineers may make exceptions for areas with contributions primarily
+    coming from one organisation and where reviewes from other organisations are
+    not possible, however, merges shall be completed by a person from a different
+    organisation.
+
+* A minimum review period of 2 days, 4 hours for trivial changes (see
+  :ref:`review_time`).
+* Hotfixes can be merged at any time after CI has passed and are excluded from
+  most of the conditions listed above.
+* All required checks are passing:
 
   * Codeowners
   * Device Tree
   * Documentation
   * Gitlint
   * Identity/Emails
   * Kconfig
-  * License
+  * License checks
```

Comment:
I would not even bother listing the individual tests. It's a required check anyway, this list changes over time, it's probably out of date anyway. Would not miss it if you were to drop it, even just "All required checks are passing" should be more than enough, sounds like saying "required check are required" :-)

Reply:
sure, not much related to this PR though, we can clean this up in a followup.